src/deadline/maya_adaptor/MayaClient/render_handlers/default_maya_handler.py

```diff
@@ -57,11 +57,19 @@ def get_camera_to_render(self, data: dict) -> list[str]:
     def get_render_layer_to_render(self, data: dict) -> Optional[str]:
         display_name = data.get("render_layer")
         if display_name:
+            # get render manager
+            filter = maya.cmds.itemFilter(byType="renderLayerManager")
+            render_manager = maya.cmds.lsThroughFilter(filter)[0]
```

Comment:
Not super familiar with Maya, can there only ever be one renderLayerManager or is [0] always the correct index?

Reply:
Good question. Looks like not only is it guaranteed that there will at least be one render layer manager, but it is guaranteed to be called "renderLayerManager". So the code can be simplified.

There can be other nodes of the same type, but they won't be the actual manager.